homeassistant/components/google/calendar.py

```diff
@@ -81,6 +83,83 @@
 SERVICE_CREATE_EVENT = "create_event"
 
 
+@dataclass(frozen=True)
+class GoogleCalendarEntityDescription(EntityDescription):
+    """Google calendar entity description."""
+
+    name: str = ""
+    entity_id: str = ""
```

Comment:
Seems weird default values. Can we omit them?

Reply:
I may be missing something but my impression is the parent class defines default values so this has to also but maybe I need another annotation... To not make a constructor or something?

Reply:
OK, `kw_only=True` was what I was missing.